node/src/types/block.rs

```diff
@@ -580,11 +666,10 @@ pub struct BlockHeader {
     body_hash: Digest,
     random_bit: bool,
     accumulated_seed: Digest,
-    era_end: Option<EraEnd>,
+    switch_block_data: Option<SwitchBlockData>,
```

Comment:
I'm not a fan of suffixing things with "_data" ... it's _all_ data. Also, now we have to describe another concept in addition to the era concept. Also, this is going to affect downstream consumers which it would be better to avoid if possible. 

I realize @afck asked for this but I would prefer that this be returned to era_end / EraEnd.

Reply:
Makes sense.

We just need a name for the other type then, the one without the validator set, containing equivocators, rewards and inactive validators. I'm really out of ideas there. `ValidatorPerformance`? :grimacing:

Reply:
`ValidatorRating`? `EraEndJudgement`? `EraReport`?

Reply:
Renamed `SwitchBlockData` to `EraEnd` and the OG `EraEnd` to `EraReport`